src/index.ts

```diff
@@ -8,7 +8,7 @@ const trimOsText = (version: string) => {
 	// Remove "Resin OS" and "Balena OS" text
 	return (
 		version
-			.replace(/(resin|balena)\sos\s+/gi, '')
+			.replace(/(resin|balena)\s*os\s*/gi, '')
```

Comment:
`?` might have been a better match than `*` but being extra defensive about white spaces makes me feel more confident that this isn't going to break by accident :+1: